php-cs-fixer

```diff
@@ -83,7 +83,7 @@ if (class_exists('Phar')) {
 
 if ($require) {
     // OK, it's not, let give Composer autoloader a try!
-    $possibleFiles = [__DIR__.'/../../autoload.php', __DIR__.'/../autoload.php', __DIR__.'/vendor/autoload.php'];
+    $possibleFiles = [__DIR__.'/vendor/autoload.php', __DIR__.'/../autoload.php', __DIR__.'/../../autoload.php'];
```

Comment:
??

Reply:
https://3v4l.org/DG4MG

Reply:
tests needed

Reply:
maybe also a in-code comment with reasoning

Reply:
if we suppress the warning than the order doesn't need to be changed does it @mvorisek ?